index.js

```diff
@@ -539,6 +539,19 @@ module.exports = async function(options) {
                     type: 'url'
                   }
                 ].concat(options.addFields || []);
+
+                options.arrangeFields = [
+                  {
+                    name: 'multisite',
+                    label: 'Multisite',
```

Comment:
I think this tab should be called "URLs," not "multisite." The former is descriptive of what you'll put there, the latter is an inside baseball term unless you're a developer.